sites/docker-compose.wayland.yml

```diff
@@ -0,0 +1,21 @@
+version: "3"
+
+services:
+    hub-be:
+        environment:
```

Comment:
Seems to be missing a lot of environment variables for database connectivity and such